timelock-api/src/main/conjure/timelock-api.yml

```diff
@@ -119,6 +121,19 @@ types:
       ConjureUnlockResponse:
         fields:
           tokens: set<ConjureLockToken>
+      ConjureRefreshLocksRequestV2:
+        fields:
+          tokens: set<ConjureLockTokenV2>
+      ConjureRefreshLocksResponseV2:
+        fields:
+          refreshedTokens: set<ConjureLockTokenV2>
+          lease: Lease
```

Comment:
Since we are doing this work, any consideration to making `Lease` serialise more compactly?

Reply:
We could, though I'm slightly less interested in this because lock refreshes are generally batched, and each request has one lease but multiple tokens. So what's in a `Lease`?:
- `LeaderTime`: consisting of an `id` which is a UUID, and a `currentTime` which is a long (and these are already correctly generated as non-wrappers)
- `Duration`: not 100% sure how this one gets serialized
I think we can come back to this if we determine they're large.